### ISSUE TITLE
Multi-Split AC showing only one unit

### ISSUE THREAD

**Issue by nurimoca** [sampsyo/hass-smartthinq#76]
Hello, I follow the instructions for this integration, but only one of the 4 units appears on HA.
Does anyone know how to fix this?

**Comment by gmusser** [comment]
How did you get even a single unit to appear? Is there anything you did, beyond the instructions?

**Comment by nurimoca** [comment]
No, I just followed the instructions. But only one of the units appear in home assistant as a climate entity.
